website/src/data/gallery.json

```diff
@@ -0,0 +1,163 @@
+[
+  {
+    "title": "AutoGen Playground",
+    "link": "https://huggingface.co/spaces/thinkall/AutoGen_Playground",
+    "description": "A space to explore the capabilities of AutoGen.",
+    "image": "default.png",
```

Comment:
It'll be good if image is not necessary.

Reply:
Good point. I can update this such that if the field is empty, it defaults to "default.png". 
Having an image is useful for visual consistency. When available, it will help the user get an idea of what the link is about even before clicking.

Reply:
Added.

Reply:
I do not find a template yet. Maybe we can hold on. I see [BoxHQ](https://boxyhq.com/) uses local image from its source code.

Reply:
The current behavior is that if there is no image added, we use a local default image `static/img/gallery/default.png`. This is what we expect correct?